### ISSUE TITLE
US::12 See events taking place at the library

### ISSUE THREAD

**Issue by algorythm** [dtugroupd/TasksIssueTracker#26]
## US::12 See events taking place at the library

> As a student, I want to see which events take place at the library at a given time, so I know when to attend certain events

## Details

If the library already published events somewhere (i.e. Facebok), we should tab in to the already existing feed, so they don't need to maintain duplicate data.

**Comment by algorythm** [comment]
We could send them a mail to ask if they post event somewhere. It would be useful information to have, before starting on this task.

**Comment by algorythm** [comment]
Removed label "stretch goal" as it replaces US::04, US::06 and US::07 (#18, #20, #21).
